### PR TITLE
MBS-10365: Convert entity lists to react-table (II)

### PR DESCRIPTION
**Pull request by reosarevok**
MBS-10365

* [ ] Artist
* [ ] Event
* [ ] Recording
* [x] Release
* [ ] Release group

### CODE REVIEW

File: root/utility/tableColumns.js [hunk 1]
```diff
@@ -294,14 +421,20 @@ export const seriesOrderingTypeColumn:
...
-
 export const subscriptionColumn:
   ColumnOptions<{+subscribed: boolean, ...}, boolean> = {
     Cell: ({cell: {value}}) => yesNo(value),
     Header: N_l('Subscribed'),
     accessor: 'subscribed',
   };
 
+export const taggerColumn:
+  ColumnOptions<{+subscribed: boolean, ...}, boolean> = {
```
Comment (mwiencek) [inline root/utility/tableColumns.js]: The `+subscribed` part of the type seems wrong.

Comment (reosarevok) [reply root/utility/tableColumns.js]: Oh, yes. I changed it to ColumnOptions<RecordingT | ReleaseT, RecordingT | ReleaseT> but I'm actually not sure what the second part should even be (I don't have an accessor at all here).

Comment (yvanzo) [reply root/utility/tableColumns.js]: `ColumnOptions<RecordingT | ReleaseT, void>` would probably be just fine.

File: root/components/list/ReleaseList.js [hunk 2]
```diff
@@ -50,192 +52,98 @@ const ReleaseList = ({
...
+  const columns = React.useMemo(
+    () => {
+      const checkboxColumn = $c.user_exists && checkboxes
+        ? defineCheckboxColumn(checkboxes)
+        : null;
+      const seriesNumberColumn = seriesItemNumbers
+        ? defineSeriesNumberColumn(seriesItemNumbers)
+        : null;
+      const nameColumn =
+        defineNameColumn<ReleaseT>(l('Release'), order, sortable, false);
```
[the remark this answers is not in the document]
Comment (reosarevok) [reply root/components/list/ReleaseList.js]: I added a comment for now (I'd be ok with changing this to a structured argument but then we should do it for all the functions, so maybe it's better to finish this first and then make that change in one go / one PR).

**Pull request merged**